src/server/pps/persist/persist.proto

```diff
@@ -52,6 +52,9 @@ message PipelineInfos {
   repeated PipelineInfo pipeline_info = 1;
 }
 
+message SubscribeNewPipelineRequest {
+}
```

Comment:
It's because I wanted to add the optimization which was to only subscribe for pipelines for a certain shard; that's when we will need this request.